diff.go

```diff
@@ -2,10 +2,13 @@ package reviewdog
 
 import (
 	"context"
+	"fmt"
 	"os/exec"
 	"sync"
 )
 
+var Y = fmt.Sprintf("test change %s")
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
missing argument for Sprintf("%s"): format reads arg 1, have only 0 args